css/style.css

```diff
@@ -0,0 +1,206 @@
+body {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+}
+
+.flex {
+  display: flex;
+}
+
+.container {
+  width: 100%;
+  flex-direction: column;
+  align-items: center;
+}
+
+/* Main Section */
+
+.main-section {
+  flex-direction: column;
+  align-items: center;
+  width: 80%;
+  background-image: url(../images/main-section-background.jpg);
+  background-size: cover;
+  background-position-y: -750px;
+  height: 650px;
+}
+
+.navbar {
+  height: 36px;
+  margin-top: 50px;
+  width: 70%;
+}
+
+.navbar button {
+  color: white;
+  cursor: pointer;
+}
+
+.normal-box-1 {
+  background: linear-gradient(to top, #6e6e6e, #4d4d4d, #6b6b6b 55%, #6e6e6e 45%, #707070, #6e6e6e);
+  width: 11%;
+  border-top-left-radius: 5px;
+  border-bottom-left-radius: 5px;
+  border: 1px gray solid;
+  text-align: center;
+}
+
+.normal-box {
+  background: linear-gradient(to top, #6e6e6e, #4d4d4d, #6b6b6b 55%, #6e6e6e 45%, #707070, #6e6e6e);
+  width: 11%;
+  border: 1px gray solid;
+}
+
+.search-box {
+  background: linear-gradient(to top, #6e6e6e, #4d4d4d, #6b6b6b 55%, #6e6e6e 45%, #707070, #6e6e6e);
+  width: 12%;
+  align-items: center;
+  border-top-right-radius: 5px;
+  border-bottom-right-radius: 5px;
+  border: 1px gray solid;
+}
```

Comment:
- [ ] You are repeating the same rules in different parts of your CSS. For good coding practices ([check this link](https://speckyboy.com/good-bad-css-practices)), follow the DRY rule (Don't repeat yourself).
![image](https://user-images.githubusercontent.com/53256139/89356769-33203f80-d684-11ea-8a2b-7776e5a73811.png)